apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -255,7 +258,8 @@ export default class MusicBlocklyWorkspace {
         const id = block.getFieldValue(TRIGGER_FIELD);
         this.compiledEvents[triggerIdToEvent(id)] = {
           code:
-            Blockly.JavaScript.blockToCode(block) + functionImplementationsCode,
+            `var __context = "${id}";\n` +
+            Blockly.JavaScript.workspaceToCode(this.workspace),
```

Comment:
My bad: this change regressed `simple2` behavior, causing all workspace code to execute when any trigger was handled, causing users to see the results of all triggers, not just the one invoked.  This is fixed with https://github.com/code-dot-org/code-dot-org/pull/61105.